app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -808,9 +808,11 @@ class PatientRepository @Inject constructor(
         ?.diagnosedWithHypertension == Answer.Yes
     if (hasHypertension) return false
 
-    val hasRequiredPrescribedDrugs = database.prescriptionDao()
-        .forPatientImmediate(patientUuid)
-        .filterNot { it.name == "Amlodipine" || it.name == "Losartan" }
+    val prescriptions = database.prescriptionDao().forPatientImmediate(patientUuid)
+    if (prescriptions.size != 2) return false
```

Comment:
Isn't the patient eligible for reassignment if they are just prescribed one medicine from the given two (Losartan and Amlodipine)?

Reply:
Turns out it's both, not one.